app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Posts</h1>
\ No newline at end of file
+<div class="posts">
+ <%= render "shared/details", user: @user %>
+  <div>
+   <% @posts.each_with_index do |post| %>
+     <%= link_to user_post_path(user_id:@user.id, id: post.id) do%>
+       <%= render "shared/post_details", post: post %>
+        <div class="comments">
+        <% if post.comments.empty? %>
+          <p>Empty</p>
+        <% else %>
+            <% @recent_comments.each do |comment| %>
```

Comment:
- [ ] After fixing the error in this view, I suggest styling the page according to the wireframe so it looks professional

|Posts View|Wireframe|
|-|-|
|![Screenshot_20230309_200804](https://user-images.githubusercontent.com/25227022/224129280-324455ca-3a8c-4ebe-9bf6-253e7e87472b.png)|![Screenshot_20230309_200903](https://user-images.githubusercontent.com/25227022/224129301-8f5afc52-bede-4e47-b4af-c1f78a96c1f1.png)|